aws_kube_params.inc

```diff
@@ -0,0 +1,26 @@
+export KUBERNETES_PROVIDER=aws
+export CLOUD_PROVIDER=${KUBERNETES_PROVIDER}
+export KUBE_AWS_INSTANCE_PREFIX=arktos
+
+# TODO: Query cidr info from VPC
+export KUBE_AWS_ZONE=us-east-2b
+export VPC_ID=vpc-0bb8ff51fd37c59b1
+export SUBNET_ID=subnet-00550372a5041fc32
+export KUBE_VPC_CIDR_BASE=10.245
+export KUBE_SUBNET_CIDR=10.245.1.0/24
+export NUM_NODES=5
+export KUBEMARK_NUM_NODES=100
+
+export AWS_IMAGE=ami-08cec7c429219e339
```

Comment:
Can you add a comment which OS version this image is for?

Reply:
i've updated the md file in the latest commit. this will be looked up from the zone to make it easier.